modules/openapi-generator/src/main/resources/swift5/libraries/alamofire/AlamofireImplementations.mustache

```diff
@@ -348,7 +348,11 @@ private var managerStore = SynchronizedDictionary<String, Alamofire.Session>()
                 }
 
                 guard let data = dataResponse.data, !data.isEmpty else {
-                    completion(.failure(ErrorResponse.error(-1, nil, dataResponse.response, DecodableRequestBuilderError.emptyDataResponse)))
+                    if T.self is ExpressibleByNilLiteral.Type {
+                        completion(.success(Response(response: dataResponse.response, body: Optional<T>.none as! T)))
```

Comment:
@Paul-Etienne you need to replace `dataResponse.response` with `dataResponse.response!`

Reply:
Oh yeah right, about that, when writing this I was wondering if I could just place this `guard` block, currently line 350, after the httpResponse is unwrapped in the next `guard` block?

That way we can use httpResponse which has already been safely unwrapped. I don't see a reason to do things in the order they're currently done, stop me if I'm wrong.

Reply:
Looks good to me 👍

Reply:
That ought to be fixed now. I hope I didn't forget anything.

Reply:
Thanks 👍 
Let's see if the CI passes.